homeassistant/components/spider/climate.py

```diff
@@ -50,6 +50,7 @@ def __init__(self, api, thermostat):
     def device_info(self) -> DeviceInfo:
         """Return the device_info of the device."""
         return DeviceInfo(
+            configuration_url="https://mijn.ithodaalderop.nl/",
```

Comment:
Since you are using the same base class in the underlying library for all devices wouldn't it be better to move all related code to a shared class (i.e. `SpiderDevice`) and make use of inheritance? 

Ref.: https://github.com/peternijssen/spiderpy/blob/master/spiderpy/devices/base.py#L4